articles/test-runner-plugins.md

```diff
@@ -0,0 +1,236 @@
+<!--
+parent: 'Developer Guide'
+toc: true
+skip : true
+authors:
+    - 'Bertrand Chevrier'
```

Comment:
So I should move the md file into the folder `articles/Test Runner Plugins`  and remove the `parent` meta ?